sourcebans-web-theme-fluent/page_servers.tpl

```diff
@@ -35,6 +35,7 @@
                             <tr id="server_{$server.sid}:desc" class="table_hide">
                                 <td colspan="8">
                                     <div class="collapse_content" id="sinfo_{$server.sid}">
+                                    <p class="responsive_show:desktop" align="center"><a href="https://www.gametracker.com/server_info/{$server.ip}:{$server.port}/"><img style="width: auto; height: auto;" src='https://cache.gametracker.com/server_info/{$server.ip}:{$server.port}/banner_560x95.png'/></a></p>
```

Comment:
Delete this. Not everyone wants this functionality.

Gametracker is very slow to load which may slow down the page loading speed